schedulezoomtalks.py

```diff
@@ -0,0 +1,146 @@
+        "contact_email": "vsf@virtualscienceforum.org",
+      },
+
+      "questions": [
+        {"field_name": "First Name", "required": True},
+        {"field_name": "Last Name", "required": True},
+        {"field_name": "Email Address", "required": True},
+        {"field_name": "Confirm Email Address", "required": True},
+        {"field_name": "Organization", "required": True},
+      ],
+
+      "custom_questions": [
+        {
+          "title": "May we contact you about future Virtual Science Forum events?",
+          "type": "single", # short or single
+          "answers": ["Yes", "No"], # only single
+          "required": true
+        },
+        {
+          "title": "How did you hear about the Virtual Science Forum?",
+          "type": "single", # short or single
+          "answers": ["Email list",
+                      "One of the organizers",
+                      "A colleague (not an organizer)",
+                      "Other"],
+          "required": true
+        },
+        {
+          "title": "Please confirm you have read the participant instructions: \
+                    http://virtualscienceforum.org/#/attendeeguide*",
+          "type": "short", # short or single
+          "required": true
+        },
+      ]
+    }
+
+    response = zoom_request(
+        requests.post,
+        f"{ZOOM_API}users/{user_id}/meetings",
+        params={"body":request_body}
+    )
+
+    return response.id, zoom_password
+
+def parse_talks(talks) -> int:
+    num_updated = 0
+    for talk in talks:
+        # If we are not processing a speakers corner talk, or if the
+        # zoom meeting id has already been set, there's nothing left to do
+        if "zoom_meeting_id" in talk or talk["event_type"] != "speakers_corner":
+            continue
+
+        # Schedule the talk
+        meeting_id, meeting_password = schedule_zoom_talk(talk)
+        # Update the talk
+        talk.get("zoom_meeting_id") = meeting_id
+        # Add this talk to researchseminars.org
+        publish_to_researchseminars(talk)
+
+        # TODO: Do something with the password
+        # do_something_with_password()
+
+        num_updated += 1
+
+    return num_updated
+
+if __name__ == "__main__":
+    # Get a handle on the repository
+    gh = github.Github(os.getenv("VSF_BOT_TOKEN"))
+    repo = gh.get_repo("virtualscienceforum/virtualscienceforum")
+
+    # Read the talks file
+    yaml = YAML()
+    try:
+        talks_data = repo.get_contents(TALKS_FILE, ref="master")
+        talks = yaml.load(StringIO(talks_data.decoded_content.decode()))
+    except github.UnknownObjectException:
+        talks_data = None
+        talks = []
+
+    # If there are talks to be parsed...
+    if len(talks) != 0:
+        # ... parse them and keep track of how many we updated
+        num_updated = parse_talks(talks)
+
+        # If we added Zoom links, we should update the file in the repo
+        if num_updated != 0:
```

Comment:
```suggestion
        # If we added Zoom links, we should update the file in the repo
        if parse_talks(talks):
```

Reply:
I do not recall where, but I do remember you very fondly advocating that it is an if *statement*, not a function ;) 
Has little to do with this suggestion, other than it triggering that memory for me. Updated